test_main.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from main import calculate_word_aabb
+from PIL import Image, ImageFont
+
+def test_testenv():
+    assert True
+
+def test_calculate_word_aabb():
+    font = ImageFont.truetype("fonts/open-sans/OpenSans-Regular.ttf", 12)
```

Comment:
Does pytest support some kind of setup and teardown for stuff like this in the future

Reply:
yes it is. Good point 👍
[https://docs.pytest.org/en/latest/fixture.html](https://docs.pytest.org/en/latest/fixture.html)